### ISSUE TITLE
Add resolution settings.

### ISSUE THREAD

**Issue by OffifialMITX** [vijai1996/screenrecorder#8]
I really like your app but there are some problems I have with it.

1. My device's screen resolution is 480 x 854 so for now I can't make videos with the native resolution. I think this shouldn't be solved by adding thousands of resolutions but by making the capture resolution adaptive.

2. You may have noticed this already, but if I switch to landscape format the output video (of course) keeps the resolution and the whole screen is being scaled down. Probably the easiest way to fix this is making a check box with a function to just don't rotate the input.

**Comment by vijai1996** [comment]
I already knew the 1st case and the problem is, the api is returning display metric including the nav bar height for some devices and for some, without it. It changes from OEM to OEM and was impossible for me to guess. The above issue is seen only on devices with nav bar. 

For the second, do you mean to prevent rotation of screen with a setting?

**Comment by OffifialMITX** [comment]
No, I mean that the view is fixed to the physical screen. It ala

**Comment by OffifialMITX** [comment]
It also may make sense to make landscape videos. I think this feauture should be easier to add.

**Comment by jhyvar** [comment]
This is an excellent and fast screen recorder. I would also like to record my videos in landscape mode.

**Comment by jhyvar** [comment]
I'm using this to record game videos but now i always have to crop them to get fullscreen videos because the game is in landscape mode but the video is recorded in portrait mode. Also would like to start recording from notification area's button

**Comment by vijai1996** [comment]
@jhyvar Open a separate issue and I'll address it

**Comment by vijai1996** [comment]
@jhyvar The orientation of the recorded video depends on the orientation the recording was started. Since the app uses android native api and its not possible to change the orientation (thus the resolution) while recording.

**Comment by OffifialMITX** [comment]
It always records in portrait mode for me.

**Comment by OffifialMITX** [comment]
Turns out I can change and tecord at 720p if I use landscape while I set it in the app, but it still records in portrait mode.

**Comment by vijai1996** [comment]
@OffifialMITX Landscape recording is something which is known for long. I will take it up as priority and do something to handle it.
